### DISCUSSION TITLE
Thumnail item bug ?

### DISCUSSION THREAD

**Discussion by 8CupCak3** [Astropilot/ValheimTooler#20] [Q&A] [answered]
![bug](https://user-images.githubusercontent.com/48679892/210031706-313f3f64-db95-482e-a156-64d38dd2c063.png)
Is there anyway to make the icon become smaller ?

**Comment by Astropilot** [comment, marked as answer]
Hi @Robert-Elias159 ,

I moved your discussion into a issue as it is more appropriate for me to follow bug reports, you can find it here: https://github.com/Astropilot/ValheimTooler/issues/21

I will also answer in this issue, thank you!
